src/client/styles/custom/custom-paper-theme.css

```diff
@@ -3539,6 +3539,7 @@ tbody.collapse.in {
   transition-timing-function: ease;
 }
 .caret {
+  font-size: 1.5em;
```

Comment:
it's not a good practice to override bootstrap classes, it's better to create a separate class for this styling